clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/agent/ClusterCachingAgent.java

```diff
@@ -74,6 +78,7 @@ public CacheResult loadData(ProviderCache providerCache) {
       nextToken = listClustersResult.getNextToken();
     } while (nextToken != null && nextToken.length() != 0);
 
+    log.info("Caching " + dataPoints.size() + " clusters in " + getAgentType());
```

Comment:
Is this a cluster in the spinnaker sense, or aws sense? please clarify

Reply:
Clarified.

It's in ECS sense of a cluster.

Reply:
I've also renamed the class to reflect this, now it's `EcsClusterCachingAgent`.